specification/paths/ServiceRates.json

```diff
@@ -114,5 +142,136 @@
         }
       }
     }
+  },
+  "post": {
+    "tags": [
+      "ServiceRates"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "organizations.manage"
+        ]
+      }
+    ],
+    "summary": "Create a new service rate.",
+    "description": "This endpoint creates a new service rate from the posted data.",
+    "requestBody": {
+      "description": "The service rate object to be created.",
+      "required": true,
+      "content": {
+        "application/vnd.api+json": {
+          "schema": {
+            "type": "object",
+            "required": [
+              "data"
+            ],
+            "additionalProperties": false,
+            "properties": {
+              "data": {
+                "allOf": [
+                  {
+                    "$ref": "#/components/schemas/ServiceRate"
+                  },
+                  {
+                    "properties": {
+                      "id": {
+                        "readOnly": true
+                      },
+                      "attributes": {
+                        "properties": {
+                          "purchase_price": {
+                            "$ref": "#/components/schemas/Price"
+                          }
+                        }
+                      },
+                      "relationships": {
+                        "properties": {
+                          "service_options": {
+                            "properties": {
+                              "data": {
+                                "items": {
```

Comment:
Don't we miss a "type": "array", above here? Or don't we need this because of inheritance.

Reply:
Inheritance